supervisor/docker/homeassistant.py

```diff
@@ -65,7 +65,7 @@ def _run(self) -> None:
             hostname=self.name,
             detach=True,
             privileged=True,
-            init=True,
+            init=False,
```

Comment:
Shouldn't we keep the init as we disable S6 with the blank entry point?

Reply:
that is for HA itself